docs/DeveloperGuide.md

```diff
@@ -355,6 +355,61 @@ Pros: Format similar to the delete command for project and task so would be easy
 Cons: Troublesome to delete since you might have to use multiple commands like list then find the `INDEX` before deleting.
 If you already know the `PROJECT_NAME` and `STAFF_NAME`, current implementation helps you to delete staff faster.
 
+## **Implementation**
+
+This section describes some noteworthy details on how certain features are implemented.
+
+### Adding staff feature
+#### Implementation
+The adding staff features is facilitated by `AddStaffCommand`. It extends the `Command` class with functionality
+to add a  `Staff` to a `Project`, both of which are provided via the `addStaff` command.
```

Comment:
I think should be `addstaff` command, small letter s since its the command word for the command.